npm-api-maker/src/commands-pool.cjs

```
@@ -144,6 +147,22 @@ module.exports = class ApiMakerCommandsPool {
     }
   }
 
+  handleErrorResponse(commandData, commandResponseData) {
+    let error
+
+    if (commandResponseData.error_type == "validation_error") {
+      const validationErrors = new ValidationErrors({
+        model: digg(commandResponseData, "model"),
+        validationErrors: digg(commandResponseData, "validation_errors")
+      })
+      error = new ValidationError(validationErrors, {response: commandResponseData})
+    } else {
+      error = new CustomError("Command error", {response: commandResponseData})
+    }
+
+    commandData.reject(new CustomError("Command failed", {response: commandResponseData}))
```

This should return the error object, right?
```suggestion
    commandData.reject(error)
```

---

That should be fixed now. I've also changed the spec since it shouldn't pass without your suggestion :-)